pmoves/docs/TAC/TAC_E2B_SANDBOX.md

```
@@ -0,0 +1,149 @@
+## Services
+
+| Service | Port | Technology | Health |
+|---------|------|-----------|--------|
+| **E2B MCP Server** | 7073 | Node 22 Alpine, MCP bridge | `GET /healthz` |
+| **E2B Surf** | 3080 | Next.js standalone, browser automation | `GET /api/health` |
+| **E2B Desktop** | 6080 | NoVNC, browser-based VNC | `GET /health` |
+| **E2B Sandbox** | 7070 | Firecracker microVM backend | `GET /health` |
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)